### DISCUSSION TITLE
集群资源共享平台 Mesos 的设计原理 · NSDI '11 - 面向信仰编程 · /papers-mesos

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#640] [Announcements]
https://draveness.me/papers-mesos/

**Comment by lukeroke** [comment]
支持作者的顶会系列，但这篇文章的标题有误：Mesos 是 NSDI'11 而非 2021，是十年前的文章。

**Comment by draveness** [comment]
> 支持作者的顶会系列，但这篇文章的标题有误：Mesos 是 NSDI'11 而非 2021，是十年前的文章。

已修复

**Comment by yuchanns** [comment]
听煎鱼说了才知道原来左大已经肉身翻到新加坡Shopee去了，之前最高产的时候竟然在字节，果真时间管理大师！

**Comment by draveness** [comment]
> 听煎鱼说了才知道原来左大已经肉身翻到新加坡Shopee去了，之前最高产的时候竟然在字节，果真时间管理大师！

最近比较懒了，写博客的意愿降低了

**Comment by tottea** [comment]
卧槽，原来在shopee sg，shopee内部有在用mesos

**Comment by draveness** [comment]
> @tottea 卧槽，原来在shopee sg，shopee内部有在用mesos

看这篇论文跟 Shopee 的 Mesos 没有任何关系

**Comment by wenlong139** [comment]
同样规模的mesos集群和k8s集群 运行同样多的进程,是mesos的cpu利用率高 还是k8s利用率高啊
